pkg/detectors/float/float.go

```diff
@@ -53,6 +53,7 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 				continue
 			}
 			req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", resMatch))
+			req.Header.Add("User-Agent", "TruffleHog3 (test@test.com)")
```

Comment:
**Nit:** I think `example@example.com` is a more secure email to attach to the user agent because it is an [IANA reserved domain](https://www.iana.org/domains/reserved) (in the off-chance the email actually gets used).